tabular/src/autogluon/tabular/task/tabular_prediction/tabular_prediction.py

```diff
@@ -57,6 +57,7 @@ def load(output_directory, verbosity=2):
     def fit(train_data,
             label,
             tuning_data=None,
+            unlabeled_data=None,
```

Comment:
lets move unlabeled_data to kwargs for the moment, since it is more advanced than things like 'time_limits' and 'presets'.

Reply:
Set 'unlabeled_data' as a kwarg instead of default param.